### DISCUSSION TITLE
Welcome to psiresp Discussions!

### DISCUSSION THREAD

**Discussion by lilyminium** [lilyminium/psiresp#43] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mino01** [comment]
Hi, I'm Mino01,
I have a PhD in Theoretical Chemistry with a focus on molecular dynamics simulation for protein–ligand binding. 
I'm looking to apply **psiresp** as part of a larger cookbook recipe for drug discovery projects.
